Tradeoffs in the persistent congestion threshold (#4215)
quicwg/base-drafts: draft-ietf-quic-recovery.md

persistent congestion, including some in response to PTO expiration, as TCP does
with Tail Loss Probes ({{RACK}}) and a Retransmission Timeout ({{?RFC5681}}).

> As the value kPersistentCongestionThreshold gets larger, the sender becomes
> increasingly unresponsive to persistent congestion in the network, which can
> result in aggressive sending into a congested network. Too small a value can
> result in a sender declaring persistent congestion unnecessarily, resulting in
> reduced throughput for the sender.

Reviewer: ```suggestion
Larger values of kPersistentCongestionThreshold cause the sender to
become less responsive to persistent congestion in the network, which can
result in aggressive sending into a congested network. Too small a value can
result in a sender declaring persistent congestion unnecessarily, resulting in
reduced throughput for the sender.
```

Reviewer 2: Like @ianswett proposal better.
